src/http/mod.rs

```diff
@@ -149,6 +153,24 @@ impl ResponseBody {
         self.inner.into_inner()
     }
 }
+impl ResponseBody {
+    pub async fn bytes(&mut self) -> Result<Vec<u8>, io::Error> {
+        let mut result = Vec::new();
+        self.read_to_end(&mut result).await?;
+        Ok(result)
+    }
+
+    pub async fn string(&mut self) -> Result<String, io::Error> {
+        let bytes = self.bytes().await?;
```

Comment:
Implement directly using `read_to_string`. That should be more efficient by catching encoding issues without reading the whole body first.

Reply:
`bytes` is intended to be used directly by users as well